starter-code/iteration-1/main.js

```diff
@@ -0,0 +1,27 @@
+// Iteration 1
+// 1. Create an array of strings
+var arrayTasks = [];
+// 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
+function addTask(tasks) {
```

Comment:
you should also declare the type of the parameter and also what you expect the function to return, like:

function addTask(tasks:string):number {